### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by DeadlyMC**
- Merge with upstream
- Remove notice file and make use of special license for matching files (Open for speculation)
- Updated readme

### CODE REVIEW

**Review by BluCobalt — APPROVED** [review]
nice

File: README.md [hunk 1]
```diff
@@ -1,26 +1,26 @@
-# fabric-meta
+# legacy-meta
 
-Fabric Meta ia a json HTTP api that can be used to query meta data about fabrics projects. It is updated every 5 mins.
+Legacy Meta is a JSON HTTP API that can be used to query meta data about Legacy-Fabric's projects. It is updated every 5 mins.
 
-I can be used by tools or launchers that wish to query version infomation about fabric.
+It can be used by tools or launchers that wish to query version information about Fabric.
```
Comment (skyrising) [inline README.md]:
```suggestion
It can be used by tools or launchers that wish to query version information about Legacy-Fabric.
```

**Review by skyrising — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (BluCobalt) [reply README.md]: What's better, `Legacy Fabric` or `Legacy-Fabric`?

Comment (skyrising) [reply README.md]: In line 3 Deadly used `Legacy-Fabric`, so I used that

Comment (BluCobalt) [reply README.md]: Ah

**Comment by DeadlyMC** [comment]
This PR now changes the copyright header to `Copyright (c) 2021 Legacy Fabric/Quilt`! Please mention any other changes before its merged

**Review by hYdos — APPROVED** [review]

**Comment by hYdos** [comment]
looks good. @BluCobalt you can merge

**Comment by BluCobalt** [comment]
> This PR now changes the copyright header to `Copyright (c) 2021 Legacy Fabric/Quilt`! Please mention any other changes before its merged

I'm still not 100% on the copyright change because we haven't incorporated any of quilt's toolchain yet. I get that we plan to, once it exists, but this seems a bit premature.

**Comment by hYdos** [comment]
> > This PR now changes the copyright header to `Copyright (c) 2021 Legacy Fabric/Quilt`! Please mention any other changes before its merged
> 
> I'm still not 100% on the copyright change because we haven't incorporated any of quilt's toolchain yet. I get that we plan to, once it exists, but this seems a bit premature.

well we will get to it in this year so it gets it out of the way for later

**Comment by BluCobalt** [comment]
i still think the copyright thing is concern but this needs to get merged 💯

**Pull request merged**